sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2463,6 +2463,178 @@ async Task<Page<DocumentHealthcareResult>> NextPageFunc(string nextLink, int? pa
 
         #endregion
 
+        #region Analyze Operation
+
+        /// <summary>
+        /// Analyze Operation enables the application to have multiple tasks including NER, PII and KPE.
+        /// For document length limits, maximum batch size, and supported text encoding, see
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits"/>.
+        /// </summary>
+        /// <param name="documents">The documents to analyze.</param>
+        /// <param name="language">The language that the document is written in.</param>
+        /// <param name="options">The additional configurable <see cref="AnalyzeOperationOptions"/> </param>
```

Comment:
For all analyze methods, this should say how in the options object is where the user can select which tasks to execute for a document

Reply:
applies everywhere else